### PR TITLE
Cleaning build scripts

### PR DESCRIPTION
**Pull request by SilleBille**
- Build scripts are cleaned to match with other components
- Step to add nuxwdog to nightly builds (https://pagure.io/dogtagpki/issue/3057)

Ticket: https://pagure.io/nuxwdog/issue/9

Signed-off-by: Dinesh Prasanth M K <dmoluguw@redhat.com>

### CODE REVIEW

File: nuxwdog.spec [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-Release:        2%{?dist}
+################################################################################
 Summary:        Watchdog server to start and stop processes, and prompt for passwords
 # The entire source code is LGPLv2 except for the perl module, which is GPL+ or Artistic
+URL:            http://www.dogtagpki.org/wiki/Nuxwdog
 License:        LGPLv2 and (GPL+ or Artistic)
-URL:            http://www.redhat.com/certificate_system
+
+Version:        1.0.5
+Release:        2%{?_timestamp}%{?_commit_id}%{?dist}
```
Comment (edewata) [inline nuxwdog.spec]: Add this line to show how to define the _phase macro if needed:
```
# global         _phase -a1
```
See https://github.com/dogtagpki/jss/blob/master/jss.spec#L11

File: nuxwdog.spec [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 
 # For epel5 and fc < 20 compatibility
 %{!?_pkgdocdir: %global _pkgdocdir %{_docdir}/%{name}-%{version}}
 
+# autosetup
+BuildRequires:    git
```
Comment (edewata) [inline nuxwdog.spec]: Nitpick: indentation

Comment (edewata) [reply nuxwdog.spec]: Also, use autosetup as in https://github.com/dogtagpki/jss/blob/master/jss.spec#L92.

**Review by edewata — COMMENTED** [review]
Please take a look at my comments. Please also check with @vakwetu about the deletion of the old build files.

**Comment by edewata** [comment]
@vakwetu, could you confirm that it's OK to remove these files? Thanks.

**Comment by SilleBille** [comment]
Tested on el7:

```
[root@<host> ~]# systemctl status pki-tomcatd-nuxwdog@pki-ca-Aug31-2018
● pki-tomcatd-nuxwdog@pki-ca-Aug31-2018.service - PKI Tomcat Server pki-ca-Aug31-2018 Started by Nuxwdog
   Loaded: loaded (/lib/systemd/system/pki-tomcatd-nuxwdog@.service; enabled; vendor preset: disabled)
   Active: active (running) since Tue 2018-09-11 14:26:12 EDT; 9s ago
  Process: 26592 ExecStart=/bin/nuxwdog -f /etc/pki/%i/nuxwdog.conf (code=exited, status=0/SUCCESS)
  Process: 26447 ExecStartPre=/usr/bin/pkidaemon start %i (code=exited, status=0/SUCCESS)
 Main PID: 26594 (nuxwdog)
   CGroup: /system.slice/system-pki\x2dtomcatd\x2dnuxwdog.slice/pki-tomcatd-nuxwdog@pki-ca-Aug31-2018.service
           ├─26594 /bin/nuxwdog -f /etc/pki/pki-ca-Aug31-2018/nuxwdog.conf
           └─26595 /usr/lib/jvm/jre-1.8.0-openjdk/bin/java -DRESTEASY_LIB=/usr/share/java/resteasy-base -Djava.library.path=/usr/lib64/nuxwdog-jni -classpath /usr/share/tomcat/bin/bootstrap.jar:/usr/share/tom...

Sep 11 14:25:24 <host> systemd[1]: Starting PKI Tomcat Server pki-ca-Aug31-2018 Started by Nuxwdog...
Sep 11 14:25:24 <host> pkidaemon[26447]: -----------------------
Sep 11 14:25:24 <host> pkidaemon[26447]: Banner is not installed
Sep 11 14:25:24 <host> pkidaemon[26447]: -----------------------
Sep 11 14:25:25 <host> pkidaemon[26447]: ----------------------
Sep 11 14:25:25 <host> pkidaemon[26447]: Enabled all subsystems
Sep 11 14:25:25 <host> pkidaemon[26447]: ----------------------
Sep 11 14:25:25 <host> pkidaemon[26447]: SUCCESS:  Successfully archived '/var/lib/pki/pki-ca-Aug31-2018/conf/ca/archives/CS.cfg.bak.20180911142525'
Sep 11 14:25:25 <host> pkidaemon[26447]: SUCCESS:  Successfully backed up '/var/lib/pki/pki-ca-Aug31-2018/conf/ca/CS.cfg.bak'
Sep 11 14:26:12 <host> systemd[1]: Started PKI Tomcat Server pki-ca-Aug31-2018 Started by Nuxwdog

[root@<host> ~]# rpm -qa nuxwdog*
nuxwdog-client-java-1.0.5-2.20180911200834.el7.x86_64
nuxwdog-1.0.5-2.20180911200834.el7.x86_64

[root@<host> ~]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.5 (Maipo)
```

File: build.sh [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+
+    # hard-code commit ID
+    commands="${commands}; s/%{?_commit_id}/${_COMMIT_ID}/g"
+
+    # hard-code phase
+    commands="${commands}; s/%{?_phase}/${_PHASE}/g"
+
+    # hard-code patch
+    if [ "$PATCH" != "" ] ; then
+        commands="${commands}; s/# Patch: tomcatjss-VERSION-RELEASE.patch/Patch: $PATCH/g"
```
Comment (edewata) [inline build.sh]: Should be "nuxwdog-VERSION-RELEASE.patch".

**Review by edewata — COMMENTED** [review]
Just one minor issue. Feel free to push.

**Pull request merged**